src/theme/components/audioPlayer.ts

```diff
@@ -0,0 +1,13 @@
+const AudioPlayer = {
+  baseStyle: {
+    invalid: {
+      backgroundColor: "ui.bg.default",
+      padding: "s",
+    },
+    helperErrorText: {
+      fontSize: "text.caption",
```

Comment:
The deisgns were wrong, but have now been fixed.  That was an older design and the font size for the helper text has changed since the design were originally put together.  That being the case, you do not need to add any styles to override the font sizes coming from the `ComponentWrapper` component.  The styles that were added here can be removed.

Reply:
Updated, thanks!